development/cpp/configuring_an_ide/code_blocks.rst

```diff
@@ -0,0 +1,129 @@
+.. _doc_configuring_an_ide_code_blocks:
+
+Code::Blocks
+============
+
+`Code::Blocks <https://codeblocks.org/>`_ is a free, open source, cross platform IDE.
+
+Creating a new project
+----------------------
+
+From Code::Blocks' main screen, click **Create a new project** or select **File > New > Project...**.
+
+.. figure:: img/code_blocks_file_new_project.png
+   :figclass: figure-w480
+   :align: center
+
+In the **New from template** window, from **Projects**, select **Empty project**, and click **Go**.
+
+.. figure:: img/code_blocks_new_empty_project.png
+   :figclass: figure-w480
+   :align: center
+
+Click Next, to pass the welcome to the new empty project wizard.
+
+.. figure:: img/code_blocks_wizard_welcome.png
+   :figclass: figure-w480
+   :align: center
+
+The project file should be created in the root of the cloned project folder. To achieve this, first, ensure that the **Project title** is the same as the folder name that Godot was cloned into. Unless you cloned the project into a folder with a different name, this will be **godot**.
```

Comment:
```suggestion
The project file should be created in the root of the cloned project folder. To achieve this, first, ensure that the **Project title** is the same as the folder name that Godot was cloned into. Unless you cloned the project into a folder with a different name, this will be ``godot``.
```